src/views/PanelViews/DepartmentViews/DepartmentCreationView/DepartmentCreationView.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { DepartmentFormView } from '../components';
+
+export function DepartmentCreationView() {
+    const handleCreation = () => {};
+
+    return (
+        <div>
+            <DepartmentFormView
+                buttonText='create'
```

Comment:
Just take a second and imagine how this line looks like for someone, besides you.
You are trying to specify params of some form. And the only two things that exist there are properties of some magic *button*... What is that button? Why is it specified here?

It should be clear from the name of the parameter what value should someone put there.

Reply:
E.g. `submitButtonHandler`